### PR TITLE
region select

### PR DESCRIPTION
**Pull request by Alinochka07**
Added a component for CIS countries. The modal by default shows Russian cities. If we click the CIS, it shows countries with addresses. Also, id from number type to string is fixed, so we can close OB-54 ticket as 'done'.

### CODE REVIEW

File: src/components/LocationSelector/CIS/CISCountriesSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+import { setCookie } from 'cookies-next';
+import { useSetSelectedEntity } from '../../../types/useSetSelectedEntity';
+import { Cookies, SEVEN_DAYS } from '../../../constants';
+import { LocationSelectorScroll } from '../../LocationSelectorScroll';
+
+interface CISCountriesSelectProps {
+  cityValue?: string;
+}
+
+export const cisRegion: Array<{
```
Comment (Alex-2kZharkov) [inline src/components/LocationSelector/CIS/CISCountriesSelect.tsx]: I would move this object to the API or to just *.constants.ts file

File: src/components/LocationSelector/CIS/CISCountriesSelect.tsx [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+  const selectCity = (value: string) => {
+    setCookie(Cookies.City, value, { expires: SEVEN_DAYS });
+    setSelectedCity(value);
+  };
+
+  const handlePickCity = (value: string) => {
+    selectCity(value);
+    window.location.reload();
+  };
+  let lastRenderedCountry = '';
```
Comment (Alex-2kZharkov) [inline src/components/LocationSelector/CIS/CISCountriesSelect.tsx]: having this `let ` variable indicates a bit strange approach. Maybe you could chat with chatgpt to come up with more clean solution?

File: src/components/LocationSelector/LocationSelector.tsx [hunk 3]
```diff
@@ -1,10 +1,24 @@
...
+
+type ClassesVariant = 'activeClass' | 'onHover';
 
 export const LocationSelector = (): ReactElement => {
+  const classes: Record<ClassesVariant, string> = {
+    activeClass: 'active drop-shadow-[1px_1px_0_orange]',
+    onHover:
+      'hover:transition-filter hover:duration-200 hover:ease hover:opacity-100 hover:drop-shadow-[1px_1px_0_orange]',
+  };
+  const [active, setActive] = useState('1');
```
Comment (Alex-2kZharkov) [inline src/components/LocationSelector/LocationSelector.tsx]: who is active? Need more descriptive variable name

Comment (Alex-2kZharkov) [reply src/components/LocationSelector/LocationSelector.tsx]: use enum when initializing useState, your teammates won't understand what is "1" and "2"

**Review by Alex-2kZharkov — CHANGES_REQUESTED** [review]

File: src/components/LocationSelector/LocationSelector.tsx (hunk 3 above)
Comment (Alinochka07) [reply src/components/LocationSelector/LocationSelector.tsx]: gave descriptive names like CIS, Russia

File: src/components/LocationSelector/CIS/CISCountriesSelect.tsx (hunk 2 above)
Comment (Alinochka07) [reply src/components/LocationSelector/CIS/CISCountriesSelect.tsx]: modernized this part.

File: src/components/LocationSelector/CIS/CISCountriesSelect.tsx (hunk 1 above)
Comment (Alinochka07) [reply src/components/LocationSelector/CIS/CISCountriesSelect.tsx]: moved to API, including Russian region

**Comment by Alinochka07** [comment]
it has been fixed and optimized.

**Comment by Alinochka07** [comment]
conflicts are fixed

**Review by Alex-2kZharkov — APPROVED** [review]

**Pull request merged**